### DISCUSSION TITLE
Access FastDDS Python library

### DISCUSSION THREAD

**Discussion by andreclerigo** [eProsima/Fast-DDS-python#112] [General]
How could I get Intellisense for the FastDDS Python library on an IDE like VSCode?

**Comment by JesusPoderoso** [comment]
Hi @andreclerigo, 
I am afraid that we are not providing any plugin such as the one you are suggesting.
After a quick search that would imply generating also an auxiliary autocompletion file (in Javascript or Json) to do so in VSCode, but it is not in our roadmap.
If you are interested in it, consider contacting our support team via email: [support@eProsima.com](mailto:support@eProsima.com)
 

As it is not a bug nor an error in the library, I am moving this ticket to the corresponding forum as per [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
